models/intel/person-vehicle-bike-detection-crossroad-yolov3-1020/description/person-vehicle-bike-detection-crossroad-1020.md

```diff
@@ -0,0 +1,97 @@
+# person-vehicle-bike-detection-crossroad-1020
+
+## Use Case and High-Level Description
+
+This is a YOLO V3 network finetuned for Person/Vehicle/Bike detection for security
+surveillance applications. Works in a variety of scenes and weather/lighting
+conditions.
+
+Yolo V3 is a real-time object detection model implemented with Keras\*
+from this [repository](https://github.com/david8862/keras-YOLOv3-model-set)
+and converted to TensorFlow\* framework.
+
+This model was pretrained on COCO\* dataset with 80 classes and then finetuned for
+Person/Vehicle/Bike detection.
+
+
+## Example
+
+![](./person-vehicle-bike-detection-crossroad-1020.png)
+
+## Specification
+
+| Metric                          | Value       |
+|---------------------------------|-------------|
+| Mean Average Precision (mAP)    | 48.89%      |
+| AP people                       | 58.94%      |
+| AP vehicles                     | 62.05%      |
+| AP bikes/motorcycles            | 25.66%      |
+| Max objects to detect           | 200         |
+| GFlops                          | 65.98       |
+| MParams                         | 61.92       |
+| Source framework                | Keras\*     |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve.
+
+Validation dataset consists of 34,757 images from various scenes and includes:
```

Comment:
I think, you shouldn't use comma as digit separator. It makes values more SI-like.

Reply:
OK

Reply:
Done